src/components/CertificationCard/CertificationCard.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react';
+import cn from 'classnames';
+import { ArrowLink } from '@app/components/ArrowLink';
+import { createBemBlockBuilder } from '@app/utils';
+
+import { Certificates } from './Certificates';
+
+import './CertificateCard.scss';
+
+const getBlocksWith = createBemBlockBuilder(['certification-card']);
+
+interface CertificationCardProps {
+  subtitle?: string;
+  shouldDisplayLink?: boolean;
+}
+
+export const CertificationCard: FC<CertificationCardProps> = ({
+  subtitle,
+  shouldDisplayLink = false,
+}) => {
+  return (
+    <div className={getBlocksWith()}>
+      <h2>ReportPortal certifications</h2>
+      {subtitle && <div className={getBlocksWith('__subtitle')}>{subtitle}</div>}
+      <div className={getBlocksWith('__certificates')}>
+        <Certificates />
+      </div>
+      {shouldDisplayLink && (
+        <ArrowLink
+          text="Learn more"
+          to="https://reportportal.io/blog/reportportal-completes-soc-2-type-ii-audit"
```

Comment:
We should not hardcode the `https://reportportal.io` as app can be deployed to other envs
```suggestion
          to="/blog/reportportal-completes-soc-2-type-ii-audit"
```

Will suggested variant work?